src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -137,32 +164,77 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
             isEqual={(a: OrganizationUser, b: OrganizationUser) => a.id === b.id}
             renderOption={(option) => renderUser(option, user, contributor)}
             displayLabel={(option) => renderUser(option, user, contributor)}
-            selectedValue={users?.find((userSel) => userSel.id === record.testingStaffUserId)}
+            selectedValue={users?.find((userSel) => userSel.id === record.withdrawnByUserId)}
             toT={(firstName: string) => ({ firstName } as OrganizationUser)}
             fullWidth={true}
             disabled={contributor}
           />
         </Grid>
         <Grid item xs={12}>
-          <Box sx={{ display: 'flex', background: '#F2F4F5', borderRadius: '16px', padding: 3, alignItems: 'center' }}>
-            <Grid item xs={12}>
-              <DatePicker
-                id='startDate'
-                label={strings.START_DATE_REQUIRED}
-                aria-label={strings.DATE}
-                value={record.startDate}
-                onChange={onChangeDate}
-              />
-            </Grid>
-            <Grid item xs={12} marginLeft={1}>
-              <TextField
-                label={strings.NUMBER_OF_SEEDS_TESTED_REQUIRED}
-                type='text'
-                onChange={onChange}
-                id='seedsTested'
-                value={record.seedsTested}
-              />
-            </Grid>
+          <Box sx={{ background: '#F2F4F5', borderRadius: '16px', padding: 3 }}>
+            <Box sx={{ display: 'flex', alignItems: 'center' }} mb={2}>
```

Comment:
can we look at the design for guidance here? i think the input elements need to align with other input elements in the modal.

<img width="326" alt="Terraware App 2022-09-16 11-28-22" src="https://user-images.githubusercontent.com/1865174/190707907-4c0b0a5a-4936-48c3-9806-f643fd73317c.png">